pkg/netp/icmp.go

```diff
@@ -71,6 +92,10 @@ func ICMPFromTypeAndCode64(icmpType, icmpCode *int64) (ICMP, error) {
 	return ICMPFromTypeAndCode(int64ToInt(icmpType), int64ToInt(icmpCode))
 }
 
+func ICMPFromTypeAndCode64WithoutValidation(icmpType, icmpCode *int64) (ICMP, error) {
```

Comment:
```suggestion
func ICMPFromTypeAndCode64WithoutRFCValidation(icmpType, icmpCode *int64) (ICMP, error) {
```
To be more consistent